addons/sourcemod/scripting/surftimer/misc.sp

```diff
@@ -4703,6 +4703,15 @@ public void TeleportToSaveloc(int client, int id)
 	TeleportEntity(client, g_fSaveLocCoords[client][id], g_fSaveLocAngle[client][id], g_fSaveLocVel[client][id]);
 }
 
+public Action DisablePrac(Handle timer, any data)//saveloc on start > startpos
+{
+	int client = GetClientFromSerial(data);
```

Comment:
Make sure that the client is still valid

Reply:
> Make sure that the client is still valid

this only gets run through client commands. there exist valid client checks there.